monkey/infection_monkey/main.py

```diff
@@ -43,7 +44,7 @@ def main():
 
     if 2 > len(sys.argv):
         return True
-
+    freeze_support() # required for multiprocessing + pyinstaller on windows
```

Comment:
PEP8 - at least 2 spaces before inline comment

Reply:
(We should automate this)

Reply:
*waves in the general direction of CI*